src/components/partner/dto/create-partner.dto.ts

```diff
@@ -41,6 +41,10 @@ export abstract class CreatePartner {
 
   @IdField({ nullable: true })
   readonly languageOfWiderCommunicationId?: IdOf<Language> | null;
+
+  @Field(() => [Language], { name: 'languagesOfConsulting', nullable: true })
```

Comment:
This is a list of IDs, not Language objects.
```suggestion
import { ID as IDType } from '@nestjs/graphql';
  @Field(() => [IDType], { nullable: true })
  @IsId({ each: true })
```